static/app/components/replays/replayController.tsx

```diff
@@ -244,14 +256,40 @@ const TimeAndScrubber = styled('div')<{isCompact: boolean}>`
       : ''}
 `;
 
+const NewTimeAndScrubber = styled('div')<{isCompact: boolean}>`
+  width: 100%;
+  display: grid;
+  grid-column-gap: ${space(1.5)};
+  grid-template-columns: max-content auto max-content;
```

Comment:
This might be a good place to use `grid-template-areas` to make something like this:

```
empty | timeline | empty
time  | scrubber | duration
```

I myself have only just started to use that in the feedback product, but it's been going well imo.

What'd we'd have is this right here:
```
display: grid;
grid-template-area:
  'space timeline space'
  'currentTime scrubber duration';
  grid-column-gap: ${space(1.5)};
  grid-template-columns: max-content auto max-content;
```

And then when we place the items, we won't need `TimeAndTimelineContainer` anymore, it'll be something like:
```
<NewTimeAndScrubber>
  <NewTime style={{gridArea: 'currentTime'>{formatTime(currentTime)}</NewTime>
  <ReplayTimeline style={{gridArea: 'timeline'}} />
  <StyledScrubber ref={elem} {...mouseTrackingProps} style={{gridArea: 'scrubber'}}>
    <PlayerScrubber />
  </StyledScrubber>
  <NewTime style={{gridArea: 'duration'}}>{durationMs ? formatTime(durationMs) : '--:--'}</NewTime>
</NewTimeAndScrubber>
```

1. We'd need double-check that `mouseTrackingProps` doesn't also have a `style` prop, i don't think it does.
2. We'd need to make sure that all these things accept a `style` prop, if they're coming from `styled()` then they will!